src/driver/unix_op.rs

```diff
@@ -0,0 +1,92 @@
+use std::io::{IoSlice, IoSliceMut};
```

Comment:
I prefer `unix::op` than `unix_op`. The former seems prettier.

Reply:
You mean a separate mod `unix` with `op.rs` in it?

Reply:
Yea.